docs/quick-start.ipynb

```diff
@@ -29,11 +29,39 @@
    "source": [
     "ds = xr.tutorial.load_dataset(\"eraint_uvz\").astype(\"float32\")\n",
     "\n",
-    "xb.plot_distribution(ds)",
+    "xb.plot_distribution(ds)\n",
```

Comment:
I think this got in here by accident. Could you undo this to the original version?